test/files/jvm/t8582.check

```diff
@@ -25,8 +25,8 @@ The module members are not in the InnerClass table of the module class (unless r
 An outer class has a InnerClass attribute for direct nested classes
  className[A1$B1] outerClassName[A1] innerName[B1] access[1]
 A nested class has an InnerClass attribute for itself (and also for its nested classes)
- className[A1$B1] outerClassName[A1] innerName[B1] access[1]
  className[A1$B1$C1] outerClassName[A1$B1] innerName[C1] access[1]
+ className[A1$B1] outerClassName[A1] innerName[B1] access[1]
```

Comment:
There's a comment somewhere saying
```
// sorting ensures nested classes are listed after their enclosing class thus satisfying the Eclipse Java compiler
```
This change seems to break that property, no?

Reply:
Declared classes are still sorted, as a referenced classes which would still satisfy that requirement. But to be conservative I'll restore the global sorting.